### DISCUSSION TITLE
TestFlight is now available 🎉

### DISCUSSION THREAD

**Discussion by arekf** [yattee/yattee#145] [Announcements]
I'm happy to announce Yattee for iOS and tvOS is now available via TestFlight. macOS builds will be added in the future as well. TestFlight will allow for easier distribution and I will also automatically receive crash logs for debugging.

Happy testing everyone!

## [Join TestFlight Beta](https://yattee.stream/beta)

**Comment by ohlala85** [comment]
can u give me a teatflight's opportunity,please

**Reply by 01SOL** [reply]
Anyway to get a testflight slot??

**Reply by arekf** [reply]
Slots are released every few days as limited number of them is remaining. You can join Discord/Matrix to get it immediately.

**Reply by ohlala85** [reply]
how can i join the discod

**Reply by ohlala85** [reply]
can u give me a link?

**Reply by arekf** [reply]
> can u give me a link?
https://yattee.stream/discord

**Reply by ohlala85** [reply]
sorry, this link seems to be invalied

**Reply by ohlala85** [reply]
this link seems to have expired

**Reply by arekf** [reply]
> this link seems to have expired

Sorry for that, not sure what happened. Use this one: https://discord.com/invite/SCr2ZfxWBB

**Reply by ohlala85** [reply]
oK thankU

**Reply by ohlala85** [reply]
btw，can u  add chinese
because my English no very good

**Reply by arekf** [reply]
> btw，can u  add chinese
> 
> because my English no very good

I am planning to create translations project in the future so people can help to bring other languages

**Reply by ohlala85** [reply]
look forward

**Comment by Duckbilled** [comment]
Sadly the TF is full, how many can actually be in such beta? Is there an expected time it'll be on the app store?

PS. would've been nice if apple had a waiting list of some sorts.

**Reply by arekf** [reply]
There are 10k TestFlight seats, all of them are used at the moment. 1500+ should be released in next 20 days.
There's no ETA for App Store yet, few months at least.

**Reply by Duckbilled** [reply]
Thank you for the reply, I’d better keep an eye open then :)

**Reply by arekf** [reply]
@Duckbilled TF is now open with 70 seats.

**Reply by Duckbilled** [reply]
Thank you! Exciting to use it. Will mostly be using it on ATV. One thing I noticed is when in a nested menu in settings, the back button actually leaves the app.
